project/accounts/views.py

```diff
@@ -188,6 +188,21 @@ def get(self, request, username=None):
         )
 
 
+class UserFollowing(LoginRequiredMixin, View):
```

Comment:
Two things are getting confused here, followers and following. Followers have subscribed to a user, while a given user can follow other users. Is the goal of this pull request to show:

A) profiles who are following this profile (followers), or
B) other profiles the current profile follows (following)?

Reply:
From my understanding of the issue #1463 I would argue that it is B). For that reason I used the `profile.following` to get to the list of profile that are following the 'current' profile, or to be more specific, profile defined in the url call of `profile/<username>/following`.

Side-note: Looking at the open issues there also is issue #1446. From my point of view, this issue was created to solve the A).

Reply:
Ok, I see that this PR follows the task definition quite closely. Sorry for my confusion.

For consistency, let's rename this view to `ProfileFollowing`.